server/views/pages/prison-register/prisonDetails.njk

```diff
@@ -34,6 +34,14 @@
       Contracted
     {% endif %}
   </p>
+  <p>
+    {% if contracted %}
+      Operator(s):
```

Comment:
Could we ever be in a position where they are set up as private/contracted but no operator is set? If so, this display would look a bit odd.

Reply:
Good point. Contracted is mandatory and defaulted false in the database. There is no way to enforce setting an operator in the database, but we should do that on Prison creation/amendment via the UI and end points (or default to PSP). We should also consider whether we really need a separate contracted flag now since this could be derived from the operator.